v5/Http/Resources/PostValueResource.php

```diff
@@ -18,13 +18,15 @@ public function toArray($request)
         $value = $this->value;
         $value_translations = $this->translations;
 
-        return [
+        $data = [
```

Comment:
Also curious, intentional or something an automated code tool did?

Reply:
Intentional, I did this so I could inspect the return output using xdebug